### ISSUE TITLE
consider creating suitable domain types

### ISSUE THREAD

**Issue by klaeufer** [sshilpika/metrics-dashboard-storage-service#2]
https://github.com/sshilpika/metrics-dashboard-storage-service/blob/master/src/main/scala/service/Boot.scala#L24

The fact that input(0), (1), and (2) occur together multiple times suggests that they belong together as some kind of structure.

**Comment by sshilpika** [comment]
You're right! I plan to have them create an object and pass the them around instead of arguments to a function. What are you're thoughts on this?
